packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -1363,6 +1363,11 @@ module.exports.external = [
             const ret = cleanUpReturn(low, high, "low", "high");
             if (ret) return ret;
           }
+        } else if (range.low || range.high) {
+          const low = getRangeLimit(range.low);
+          const high = getRangeLimit(range.high);
+          const ret = cleanUpReturn(low, high, "low", "high");
```

Comment:
this did not work for one of the most common dosage range structures :(

Reply:
Can we figure this out so this patch goes out?